lib/active_fedora/file.rb

```diff
@@ -176,6 +170,12 @@ def prefix(path)
 
     private
 
+      def create_or_update(*options)
+        super.tap do
+          metadata.save if metadata.changed?
```

Comment:
(Without looking), should this be `#save!` and/or do we need some error handling logic if `#save` fails?